### ISSUE TITLE
💄 Home page - Accessibility improvements

### ISSUE THREAD

**Issue by andrewwaltosssw** [tinacms/tina.io#3142]
### 🩸 Pain

Wave and Axe tools both report accessibility issues

### 💡 Suggested Solution

Fix them using semantic html

### 💼 Tasks 

- [ ] Run Wave extension https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh and note issues
- [ ] Run axe extension https://chromewebstore.google.com/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd and note issues
- [ ] Fix the issues

### ✅ Acceptance Criteria 

Accessibility tool scan on pages returns no issues

### 📸 Screenshots 

![Image](https://github.com/user-attachments/assets/b07992cb-d5fb-4024-b1ac-7e91807ce16e)
Figure: axe devtool scan result

![Image](https://github.com/user-attachments/assets/d6e4c257-267a-4d21-9465-09fe12414aa4)
Figure: Wave browser tool scan

Cc: @Marxoz

**Comment by andrewwaltosssw** [comment]
cc @jaydenalchin and @sambwagner in case you have some suggestions as well

**Comment by jaydenalchin** [comment]
Thanks @andrewwaltosssw I also ran Siteimprove which returned 12 key a11y issues - those tagged A or AA are important to fix - looks like most of the issues relate to missing aria-labels on link text or semantic headings. 

![Image](https://github.com/user-attachments/assets/3ea3e035-3430-47cb-b03d-ea507272133c)
**Figure: Accessibility report**

**Comment by isaaclombardssw** [comment]
Yeah really good to see these ones flagged.
